src/app/api/admin/services/externalLogin/route.ts

```diff
@@ -1,7 +1,13 @@
+import { z } from 'zod';
 import db from '@/db';
 import { NextRequest, NextResponse } from 'next/server';
 import bcrypt from 'bcrypt';
 
+const loginSchema = z.object({
+  email: z.string().email(),
```

Comment:
This is not necessarily just a email it can be any string or number

Reply:
done